docs/reference/concepts/deployments.md

```diff
@@ -32,6 +32,8 @@ For more information, see the [comprehensive worker configuration documentation]
 
 ## Local deployment
 
+Before starting a local deployment, you can set a custom snapshot as your deployment's launch snapshot. To do so select **SpatialOS** > **GDK tools configuration** and then select the file on the **Selected snapshot path** option.
```

Comment:
I'd maybe put this in a header just above `Start a standalone client worker`. 

Then the flow is:

1. How to start a deployment
2. How to configure that deployment
3. How to run a standalone client against that deployment.

Thoughts?

Reply:
You mean between the **Note** and the `Start a standalone client worker` ?

Reply:
Yeah exactly